### PR TITLE
Item 5185: Zip Files for mass spec data

### PR DESCRIPTION
**Pull request by labkey-ankurj**

### CODE REVIEW

File: internal/webapp/internal/FileDrop/FileDrop.js [hunk 1]
```diff
@@ -223,6 +224,13 @@ LABKEY.internal.FileDrop = new function () {
                 if (!e.dataTransfer) {
                     return;
                 }
+                //use colon(:) as file separator in case of firefox
+                if(window.navigator.userAgent.toLowerCase().indexOf("firefox") > -1) {
```
Comment (cnathe) [inline internal/webapp/internal/FileDrop/FileDrop.js]: @labkey-ankurj 
The comment above this if statement doesn't match.
Also, couldn't you just set the isFirefox variable directly?
var isFirefox = window.navigator.userAgent.toLowerCase().indexOf("firefox") > -1;

File: internal/webapp/internal/ZipLoad/ZipLoad.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ LABKEY.internal.ZipLoad = new function () {
     var dirPatterns;
     var dropZone;
     var filePattern =new RegExp(".*\\.zipme$");
+    var firefox;
```
Comment (cnathe) [inline internal/webapp/internal/ZipLoad/ZipLoad.js]: @labkey-ankurj 
Inside of this ZipLoad.js, this variable is more about whether we expect all files to be read at once or if we need to batch them. So can we rename this variable accordingly (maybe something like: checkReadEntriesByBatch)?

Comment (labkey-ankurj) [reply internal/webapp/internal/ZipLoad/ZipLoad.js]: changed locally

File: internal/webapp/internal/FileDrop/FileDrop.js (hunk 1 above)
Comment (labkey-ankurj) [reply internal/webapp/internal/FileDrop/FileDrop.js]: oops, missed that comment. changed locally

**Pull request merged**